lib/https.c

```diff
@@ -349,6 +350,30 @@ _establish_connection(struct https_request * const req,
 #endif /* HAVE_GETADDRINFO */
 }
 
+/* Provide implementations for HMAC_CTX_new and HMAC_CTX_free when
+ * building for OpenSSL versions older than 1.1.0
+ */
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
+static
+HMAC_CTX *HMAC_CTX_new(void)
```

Comment:
Hey Evan, I'm about good to pull this in. But I think I miscommunicated what I wanted the style to look like here.
I was hoping this line to look more like
```
static HMAC_CTX *
HMAC_CTX_new(void)
```
Thanks for bearing with me. Once we make this change I think we're good to go.